api/v1alpha1/types.go

```diff
@@ -43,6 +44,11 @@ type ResourceGraphDefinitionSpec struct {
 	//
 	// +kubebuilder:validation:Optional
 	DefaultServiceAccounts map[string]string `json:"defaultServiceAccounts,omitempty"`
+	// AdditionalPrinterColumns defines additional printer columns that will
```

Comment:
Worth calling out that setting this overrides the default printer columns, and if the user wants to retain the default columns they will need to specify them here explicitly?

Reply:
@matthchr, I just added the comment you proposed. Let me know if there is anything else